### ISSUE TITLE
Access the collection title property

### ISSUE THREAD

**Issue by zstrad44** [ahl389/shopify-buy-sdk-example#4]
Hey ahl389, 

Great solution, this pretty much fits exactly what I was looking for.  

One question I do have is if there is a way to access the collection title, so that I can dynamically insert where needed. 

For example: var selectedCollection = product.selectedCollection 

That way I can create a current-collection class, or a just append with the set var. 

I couldn't find this in the documentation, but I may have missed it. Thanks!

**Comment by ahl389** [comment]
hey @zstrad44 thanks for the suggestion - I know I'm a few weeks late, but I just updated the code.  Note, there was a slight change in the HTML to add a '.collection-title' class to the h2 holding the collection title, as well as the new JS function to fetch it. :)

**Comment by zstrad44** [comment]
Nice fix @ahl389 . I actually ended up doing something similar after looking through the SDK docs a little more in depth.  I think I may like your approach a little better, so I may copy.  

Thanks again for the collection fix, I was able to create my site thanks to your help: [My SDK Store](http://supplies.ims-trident.com). 

It's much improved thanks to your collection fix, although it is still a work in progress, now I'm working on displaying and sorting multiple collection titles, and images on a single page..
